yb-voyager/cmd/importSchema.go

```diff
@@ -200,9 +208,16 @@ func importSchema() error {
 			return false
 		}
 		skipFn := isSkipStatement
+
 		err = importSchemaInternal(exportDir, objectList, skipFn)
 		if err != nil {
-			return fmt.Errorf("failed to import schema for various objects: %s", err) // object list is the static list of object types
+			if errors.As(err, &execDDLError) {
```

Comment:
Shouldn't we also check here if schema analysis report is generated or not? 
One potential improvement:  it should probably be irrespective of whether the schema analysis report was generated in this run or manually before.